src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Filters/DateFiltersTests.cs

```diff
@@ -90,22 +131,84 @@ public static IEnumerable<object[]> GetInvalidTimeZoneHandling()
             yield return new object[] { @"19701231115959+0600", "abc" };
         }
 
+        public static IEnumerable<object[]> GetInvalidDataForAddSeconds()
+        {
+            yield return new object[] { @"20badInput" };
+            yield return new object[] { @"20140130080051--0500" };
+            yield return new object[] { @"2014.051-0500" };
+            yield return new object[] { @"20140130080051123+0500" };
+            yield return new object[] { @"20201" };
+            yield return new object[] { @"2020060" };
+            yield return new object[] { @"1970-01-01T00:01:00" };
+            yield return new object[] { @"1970-01-01T00:01" };
+            yield return new object[] { @"2001-01T" };
```

Comment:
Can we try `null` and `string.empty`? Do they belong to invalid cases?

Reply:
Null string.empty are valid input (they are included in the valid data tests) and will return null or string.empty.